src/main/java/hw_5/Task1/MyClass/Person.java

```diff
@@ -0,0 +1,37 @@
+package hw_5.Task1.MyClass;
+
+public class Person {
+    private String name;
+    private String surname;
+    private int age;
+
+    // Конструктор
```

Comment:
Comment not in English. Fix in all places please

Reply:
fixed